configs/mot/ocsort/metafile.yml

```diff
@@ -0,0 +1,39 @@
+Collections:
+  - Name: OCSORT
+    Metadata:
+      Training Techniques:
+        - SGD with Momentum
+      Training Resources: 8x V100 GPUs
+      Architecture:
+        - YOLOX
+    Paper:
+        URL: https://arxiv.org/abs/2203.14360
+        Title: Observation-Centric SORT Rethinking SORT for Robust Multi-Object Tracking
+    README: configs/mot/ocsort/README.md
+
+Models:
+  - Name: ocsort_yolox_x_crowdhuman_mot17-private-half
+    In Collection: OCSORT
+    Config: configs/mot/ocsort/ocsort_yolox_x_crowdhuman_mot17-private-half.py
+    Metadata:
+      Training Data: CrowdHuman + MOT17-half-train
+    Results:
+      - Task: Multiple Object Tracking
+        Dataset: MOT17-half-val
+        Metrics:
+          MOTA: 78.6
+          IDF1: 79.2
+    # Weights: https://download.openmmlab.com/mmtracking/mot/bytetrack/bytetrack_yolox_x/bytetrack_yolox_x_crowdhuman_mot17-private-half_20211218_205500-1985c9f0.pth
```

Comment:
Update the weights link.

Reply:
For my test, I directly use the YOLOX detector from ByteTrack. Here, also considering I have no permission to rename and upload weights, I just use the detector weights link from Bytetrack.